### PR TITLE
Update W2L for continuous processing

### PR DESCRIPTION
**Pull request by chrismbirmingham**
Includes a w2l file that listens directly to the microphone as well as one that listens to the audio topic (as apposed to the talking topic).

### CODE REVIEW

File: dockerfiles/dev/harmoni-pc/dockerfile [hunk 1]
```diff
@@ -33,6 +38,10 @@ RUN git -C src clone \
 
 RUN ln -s /usr/lib/x86_64-linux-gnu/libboost_python-py35.so /usr/lib/x86_64-linux-gnu/libboost_python3.so
 
+RUN git -C src clone \
+      -b master \
+      https://github.com/ros-drivers/audio_common.git
+
```
Comment (RMichaelSwan) [inline dockerfiles/dev/harmoni-pc/dockerfile:44]: This is probably not needed. I believe audio common comes with a regular ros install. If not included, you could always do: `apt-get install ros-kinetic-audio-common`

**Review by RMichaelSwan — CHANGES_REQUESTED** [review]
Good to merge once the audio common thing is fixed. I just realized I put most of my comments for this content in the other review, so maybe check there as well.

File: dockerfiles/dev/harmoni-pc/dockerfile (hunk 1 above)
Comment (chrismbirmingham) [reply dockerfiles/dev/harmoni-pc/dockerfile:44]: Thanks, I think this got dragged in while trying to fix audio issues. Will remove

**Comment by chrismbirmingham** [comment]
This got subsumed by the dialogue pattern branch

**Pull request closed without merging**